com.stremio.Stremio.yml

```diff
@@ -1,68 +1,52 @@
 app-id: com.stremio.Stremio
-runtime: org.freedesktop.Platform
-runtime-version: '19.08'
-branch: stable
-sdk: org.freedesktop.Sdk
-base: org.electronjs.Electron2.BaseApp
-base-version: stable
-command: stremio-wrapper
+runtime: org.kde.Platform
+runtime-version: "5.13"
+sdk: org.kde.Sdk
+command: stremio
+base: "io.qt.qtwebengine.BaseApp"
+base-version: "5.13"
```

Comment:
Can we switch to 5.12 for now?

Reply:
Yes, I just didn't had the time to update it ^^

Reply:
haha no problem, i could add a commit, but this would be to your branch? If not once you update it, what is stopping this from going to flathub?

Reply:
Upstream being involved in that; we strongly recommend upstream involvement for any new application on Flathub now, so even if it's ready, upstream has to submit/or want to co-maintain it